app/views/check_answers/_telephone.html.erb

```diff
@@ -7,8 +7,12 @@
   </dd>
   <dd class="govuk-summary-list__actions">
     <% if defined?(link) %>
```

Comment:
This section could be simplified to:

```
<% if defined?(link) && link == Equivalent::LINK %>
        <%= show_link("#{link}" + 'overseas_candidate') %>
<% else %>
        <%= show_link("#{link}" + 'overseas_telephone') %>
<% end %>
```

Also, one for another PR but it's not very clear what step `overseas_candidate` is - you have to dig up to the parent model:

```
OverseasCandidate < Callback
```

^ this inheritance would be clearer if the model was something like `OverseasCallback` or `OverseasCandidateCallback`

Reply:
Not sure that works, the else clause at line 15 needs to cover the 'no link' case ?

Reply:
![image](https://user-images.githubusercontent.com/4527528/90373117-5924e700-e069-11ea-8414-5eb2f46776fc.png)

Reply:
Ah you're right, I misread that - thought lines 12 and 15 were the same